workspaces/local-cli/package.json

```diff
@@ -45,8 +46,10 @@
     "@types/mocha": "^5",
     "@types/node": "^10",
     "@types/react-dev-utils": "^9.0.2",
+    "@types/url-join": "^4.0.0",
     "@types/uuid": "^7.0.3",
     "chai": "^4",
+    "dotenv": "^8.2.0",
```

Comment:
These should be regular dependencies right?

Reply:
You're right!